config/locales/client.en.yml

```diff
@@ -3304,6 +3304,12 @@ en:
         instructions:
           one: "Please choose the topic you'd like to move that post to."
           other: "Please choose the topic you'd like to move those <b>%{count}</b> posts to."
+        merge_type:
+          title:
+            one: "Merge this post..."
+            other: "Merge these %{count} posts..."
+          sequential: "as one sequential block at the bottom of the topic"
+          chronological: "in time order throughout the topic"
```

Comment:
The wording for these options is difficult to get right (and will likely be hard for translators as well). What do you think about changing these to: 

- append posts to the end of the topic
- insert in chronological order

Another option may be to provide the chronological order as an advanced checkbox in the existing modal. Something like this, maybe? 

<img width="500" alt="image" src="https://user-images.githubusercontent.com/368961/236248571-fe863828-2b97-4bba-9726-d5b28f9c335a.png">

Reply:
@pmusaraj I like the "advanced checkbox" suggestion, my 1st version was just a radio on this existing modal but I then changed it to be as close as possible to the requirements Jamie provided.

Reply:
Resolving both conversations, chronological order is now a checkbox on the existing modal